### DISCUSSION TITLE
Vector tiles vineyards demodataset not loading in Openlayers

### DISCUSSION THREAD

**Discussion by codehub-rony** [ldproxy/ldproxy#1173] [API] [answered]
I'm trying to add the Vineyards Vector Tiles using the [OGC Vector Tile source](https://openlayers.org/en/latest/examples/ogc-vector-tiles.html) in OpenLayers. I get a 404 http response for [tilematrixsets url](https://demo.ldproxy.net/vineyards/tileMatrixSets/WebMercatorQuad).

I tested it the same code with  a vectortile URL from a ldproxy instancethat I'm running locally, and I don't have that issue there. The tilematrixset url returns a page (see image below) and the data loads fine.

<img width="803" alt="image" src="https://github.com/interactive-instruments/ldproxy/assets/10270049/8f6e2932-5a2e-4872-85ff-6476cb9e08ab">

Can this be an issue with the configuration of the vineyards dataset? I have added the data to Openlayers before and it used to work.

**Comment by cportele** [comment, marked as answer]
@codehub-rony - Thanks for this. I have fixed the 404 issue in the demo.ldproxy.net APIs. The reason for the 404 was that originally the tile matrix sets were resources in the TILES building block, but they have been moved to their own TILE_MATRIX_SETS building block, which was not enabled for the vineyards APIs. Change: https://github.com/ldproxy/demo/commit/0b3e05e7505f368693b94ba828f44124e3fda164.
